app/views/directives/bind-service/select-application.html

```diff
@@ -0,0 +1,35 @@
+<h3 class="mar-top-none">Select an application to bind to service <strong>{{ctrl.serviceClasses[ctrl.target.spec.serviceClassName].osbMetadata.displayName || ctrl.target.spec.serviceClassName}}</strong></h3>
```

Comment:
> {{ctrl.serviceClasses[ctrl.target.spec.serviceClassName].osbMetadata.displayName || ctrl.target.spec.serviceClassName}}

Probably worth adding a `serviceClassDisplayName` filter at some point. Doesn't have to be in this PR. It might make sense in common.